aspnetcore/tutorials/razor-pages/razor-pages-start.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Getting started with Razor Pages on ASP.NET Core
+author: rick-anderson
+description: Getting started with Razor Pages on ASP.NET Core
+keywords: ASP.NET Core,Razor Pages, Razor, MVC
+ms.author: riande
+manager: wpickett
+ms.date: 7/27/2017
+ms.topic: get-started-article
+ms.technology: aspnet
+ms.prod: asp.net-core
+uid: tutorials/razor-pages/razor-pages-start
+---
+# Getting started with Razor Pages on ASP.NET Core
+
+By [Rick Anderson](https://twitter.com/RickAndMSFT)
+
+This tutorial will teach you the basics of building an ASP.NET Core Razor Pages web app. We recommend you complete [Introduction to Razor Pages](xref:mvc/razor-pages/index) before starting this tutorial.
+
+## Prerequisites
+
+* [.NET Core 2.0 SDK](https://www.microsoft.com/net/core)
+* [Visual Studio 2017 Preview version 15.3](https://www.visualstudio.com/vs/preview/)
+
+## Create a Razor web app
+
+* From the Visual Studio **File** menu, select **New > Project**.
+* Create a new ASP.NET Core Web Application. Name the project **RazorPagesMovie**. It's important to name the project *RazorPagesMovie* so the namespaces will match when you copy/paste code.
+ ![new ASP.NET Core Web Application](../../mvc/razor-pages/index/_static/np.png)
+* Select **ASP.NET Core 2.0** in the dropdown, and then select **Web Application**.
+ ![Web Application (Razor Pages)](../../mvc/razor-pages/index/_static/np2.png)
+
+The Visual Studio template creates a starter project:
+
+![Solution Explorer](razor-pages-start/_static/se.png)
+
+Press **F5** to run the app in debug mode or **Ctrl-F5** in non-debug mode.
+![Home or Index page](razor-pages-start/_static/home.png)
+
+* Visual Studio starts [IIS Express](http://www.iis.net/learn/extensions/introduction-to-iis-express/iis-express-overview) and runs your app. The address bar shows `localhost:port#` and not something like `example.com`. That's because `localhost` is the standard hostname for your local computer. Localhost only serves web requests from the local computer. When Visual Studio creates a web project, a random port is used for the web server. In the preceding image, the port number is 5000. When you run the app, you'll see a different port number.
+* Launching the app with **Ctrl+F5** (non-debug mode) allows you to make code changes, save the file, refresh the browser, and see the code changes. Many developers prefer to use non-debug mode to quickly launch the app and view changes.
+
+The default template creates **RazorPagesMovie**, **Home**, **About** and **Contact** links. Depending on the size of your browser, you might need to click the navigation icon to show the links.
```

Comment:
~~Depending on the size of your browser~~
Depending on the size of your browser window